src/content/docs/apis/nerdgraph/examples/nerdgraph-manage-users.mdx

```diff
@@ -1,21 +1,20 @@
 ---
-title: "NerdGraph tutorial: Manage users"
+title: "NerdGraph tutorial: Create and manage users"
 tags:
   - APIs
   - NerdGraph
-metaDescription: "How to use New Relic's NerdGraph API to manage users and return user information."
+metaDescription: "How to use New Relic's NerdGraph API to create and manage users, and view user information."
 ---
 
-You can use our [NerdGraph API](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph) to view and manage users. You can edit their user type and email address, and return other types of user information. 
+You can use our [NerdGraph API](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph) to view and manage users. You can add and delete users, edit users' user type and email address, and return other types of user information. 
```

Comment:
I think you can remove the "user" after "users'" unless there is a specific reason for the repetition.

Reply:
Actually nevermind, it just read a little weird but I see why it's necessary.